app/src/main/java/com/simple/demotest/MainActivity.java

```diff
@@ -0,0 +1,34 @@
+package com.simple.demotest;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    Button mClickMeButton;
+    TextView mHelloWorldTextView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        mClickMeButton = (Button) findViewById(R.id.btn_click_me);
+        mHelloWorldTextView = (TextView) findViewById(R.id.tv_hello_world);
+
+        mClickMeButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent().setClassName(getBaseContext(), "com.simple.demotest.Main2Activity");
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/demotest/MainActivity.java L26

Line is longer than 100 characters (found 113).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*